jang_seeder_wheel.scad

```diff
@@ -116,11 +117,33 @@ module jangSeederWheel(
       for(row = [0 : 1 : seed_rows - 1]) {
         rotate([0, 0, theta + (360 / seed_count * row)])
           translate([wheel_dia / 2, 0, wheel_width * (row + 0.5) / (seed_rows)])
-            if(seed_shape == "v") {
+            // Conical 'v' shap
+            if(seed_shape == "v") { e
```

Comment:
Please add these new values to the enum in `customizer.scad`

Reply:
Ah, got it. Added